### PR TITLE
Unity

### PR DESCRIPTION
**Pull request by gvn**
# DON'T MERGE - WORK IN PROGRESS

### CODE REVIEW

File: app/networkapi/settings.py [hunk 1]
```diff
@@ -303,9 +303,13 @@
 
 STATIC_URL = '/static/'
 
-STATICFILES_DIRS = (
+WHITENOISE_ROOT = app('frontend')
+WHITENOISE_STATIC_PREFIX = '/'
+WHITENOISE_INDEX_FILE = True
```
Comment (Pomax) [inline app/networkapi/settings.py]: hurray for Whitenoise 4

**Review by Pomax — APPROVED** [review]
I heartily endorse this solution, but we already knew this.

File: app/networkapi/settings.py (hunk 1 above)
Comment (gvn) [inline app/networkapi/settings.py]: 🤘 🤘

**Comment by gvn** [comment]
Note: This is now going to fail CI because of `requirements.txt` being removed (in favor of putting it into the root of `network`).

**Comment by gvn** [comment]
No further commits should be made to this branch.

Commit #b019cd13c01fc4635206ca1df03e5f94258250ed was manually copied over to the `unity` branch of `network` in [this commit](https://github.com/mozilla/network/pull/599/commits/ad9259625bc6c38d710b4656295d0e840f3c8cea).

**Pull request closed without merging**